### PR TITLE
Update for improved driving stress control

### PR DESCRIPTION
**Pull request by Gubner**
Updates to improve control of driving stress. Control by vehicle class for both buckled and unbuckled speeds. More control for Police and EMS driving stress added as well.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
- Does your code fit the style guidelines? Yes
- Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -882,13 +882,15 @@ CreateThread(function() -- Speeding
             local ped = PlayerPedId()
             if IsPedInAnyVehicle(ped, false) then
                 local speed = GetEntitySpeed(GetVehiclePedIsIn(ped, false)) * speedMultiplier
-                local stressSpeed = seatbeltOn and config.MinimumSpeed or config.MinimumSpeedUnbuckled
+		local vehicle = GetVehiclePedIsIn(ped, false)
+		local vehicleClass = GetVehicleClass(vehicle)
```
Comment (BerkieBb) [inline client.lua]: Please format both lines correctly so they align with the rest of the code

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client.lua (hunk 1 above)
Comment (Gubner) [reply client.lua]: I updated by pasting white space from line 884. Did that resolve the issue?

Comment (BerkieBb) [reply client.lua]: It did

**Comment by Maxryodric** [comment]
it was more interesting to check for the presence of a helmet, if you want an extreme ride, put on a helmet, it would be useful for motorcycles

**Comment by next-game** [comment]
> it was more interesting to check for the presence of a helmet, if you want an extreme ride, put on a helmet, it would be useful for motorcycles

I also think so, why complicate it with classes of cars, if a car of another class we will feel differently? there is a racing belt, only a racing **helmet** is missing

**Pull request closed without merging**